packages/db/lib/gen-migration.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import { stat } from 'fs/promises'
+import path from 'path'
+import loadConfig from './load-config.mjs'
+
+async function generateMigration (_args) {
+  const { configManager } = await loadConfig({}, _args)
+  const migrationsDir = configManager.current.migrations.dir
+
+  // Check migrations directory exists
+  await checkMigrationsDirectoryExists(migrationsDir)
+
+  try {
+    await configManager.parseAndValidate()
+    const files = await fs.promises.readdir(migrationsDir)
+
+    const nextMigrationId = getNextMigrationId(files)
+
+    const nextDoMigrationName = `${nextMigrationId.toString().padStart(3, '0')}.do.sql`
+    await generateMigrationFile(migrationsDir, nextDoMigrationName)
+
+    const nextUndoMigrationName = `${nextMigrationId.toString().padStart(3, '0')}.undo.sql`
+    await generateMigrationFile(migrationsDir, nextUndoMigrationName)
+  } catch (err) {
+    console.error(err.message)
+    process.exit(1)
+  }
+}
+
+function getNextMigrationId (migrationFileNames) {
+  if (migrationFileNames.length === 0) {
+    return 1
+  }
+  const lastMigrationFileName = migrationFileNames.pop()
+
+  const migrationFileFormat = path.extname(lastMigrationFileName)
+  if (migrationFileFormat !== '.sql') {
+    throw new Error(`Previous migration has wrong format: ${migrationFileFormat}`)
+  }
+
+  const chunks = path.basename(lastMigrationFileName).split('.')
+  const migrationId = parseInt(chunks[0])
+  if (isNaN(migrationId)) {
+    throw new Error('Previous migration has corrupted name')
+  }
+
+  const nextId = migrationId + 1
+  return nextId
+}
+
+async function generateMigrationFile (migrationDir, id) {
+  const newMigrationFilePath = path.join(migrationDir, id)
+  await fs.promises.appendFile(newMigrationFilePath, '')
```

Comment:
Why do you use `appendFile` and not `writeFile`? Is there a reason for that?

Reply:
Personal preference, I can use writeFile if it is preffered.

Reply:
writeFile and appendFile have different meaning. You should only use appendFile if you intend to append something to the file.

Reply:
yes, thanks

Reply:
Yep, done